src/main/java/com/conveyal/datatools/manager/models/Deployment.java

```diff
@@ -453,6 +492,16 @@ private <O extends Serializable> String writeToString(O object) {
     public byte[] generateRouterConfig() throws IOException {
         Project project = this.parentProject();
 
+        if (this.customRouterConfigUrl != null) {
+            try {
+                // TODO: should Mongo be updated here?
```

Comment:
Should this comment be added above the line for `this.customBuildConfig` as well?

Reply:
If not here, where will it be updated?

Reply:
Well I think in this line instead of updating the local instantiated Deployment we'd update Mongo